molecule/default/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: molecule/default/yamllint.yml
+platforms:
+  - name: centos6
+    image: centos:6
+    privileged: true
+  - name: centos7
+    image: vcrhonek/centos7-systemd
```

Comment:
Please add the image the to linuxsystemroles docker organization and use it from there.